Lib/pdb.py

```diff
@@ -491,6 +491,10 @@ def interaction(self, frame, tb_or_exc):
                 Pdb._previous_sigint_handler = None
 
         _chained_exceptions, tb = self._get_tb_and_exceptions(tb_or_exc)
+        if not _chained_exceptions and isinstance(tb_or_exc, BaseException):
+            raise ValueError(
+                "A valid traceback must be passed if no exception is being handled"
+            )
```

Comment:
```suggestion
        if not _chained_exceptions and isinstance(tb_or_exc, BaseException):
            raise ValueError(
                "A valid traceback must be passed if no exception is being handled"
            )
```

This is supposed to catch this reproducer

```
>>> import sys
>>> sys.last_exc = Exception()
>>> import pdb
>>> sys.last_exc = Exception()
>>> pdb.pm()
```

It's contrived, but as it is already handled in `pdb.post_mortem()`, I assume there are case where this can happen in production.